app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function ($filter,
+    $q,
+    $uibModal,
+    $rootScope,
+    APIService,
+    DataService,
+    Logger,
+    NotificationsService) {
+
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+    var remove = function (apiObject) {
+      if (!$rootScope.AEROGEAR_MOBILE_ENABLED) {
+        Logger("mobile is not enabled");
+        return;
+      }
+      var context = {
+        namespace: _.get(apiObject, 'metadata.namespace')
+      };
+      NotificationsService.hideNotification("remove-mobileclient-error");
+      DataService.delete(mobileclientVersion, apiObject.metadata.name, context, {
+        gracePeriodSeconds: 0
+      }).then(
+        // success
+        _.noop,
+        // failure
+        function (err) {
+          NotificationsService.addNotification({
+            id: "remove-mobileclient-error",
+            type: "error",
+            message: "An error occurred while deleting the mobile client" + _.get(apiObject, 'metadata.name') + ".",
+            details: $filter('getErrorDetails')(err)
+          });
+          Logger("An error occurred while deleting mobileclient " + _.get(apiObject, 'metadata.name') + ".", err);
+        });
+    };
+
+
+    var deleteClient = function (apiObject) {
```

Comment:
Any reason you're not using our standard delete dialog?

Reply:
As above I was looking at how it was done for service instances service and tried to follow that. Is there an example you can point me to that is more standard

Reply:
You should be able to use the `delete-link` directive and remove a lot of this code. Here's an example:

https://github.com/openshift/origin-web-console/blob/master/app/views/browse/pod.html#L25-L31

Reply:
Thanks will update to use that

Reply:
based on that I think I can remove the mobile service for now too